autoreject/autoreject.py

```diff
@@ -869,9 +868,26 @@ def fit(self, epochs):
                 epochs_interp, interp_channels=interp_channels,
                 verbose=self.verbose)
 
-            for fold, (train, test) in enumerate(_pbar(self.cv, desc='Fold',
-                                                 position=3,
-                                                 verbose=self.verbose)):
+            X = epochs.get_data()[:, self.picks]
+
+            # Hack to allow len(self.cv.split(X)) as ProgressBar
+            # assumes an iterable whereas self.cv.split(X) is a
+            # generator
```

Comment:
@agramfort I had to do this hack. Do you have any better suggestion? Otherwise, this PR is good to go